server/controllers/content.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable sort-keys */
+/* eslint-disable consistent-return */
+const Content = require("../models/Content");
+
+// Create and Save a new Content
+exports.create = (req, res) => {
```

Comment:
go through each and every one of these function and make each one an `async/await` function, then use the `try/catch` pattern to catch your errors.